routeros/resource_interface_veth.go

```diff
@@ -33,9 +33,19 @@ func ResourceInterfaceVeth() *schema.Resource {
 		KeyDisabled: PropDisabledRw,
 		"gateway": {
 			Type:         schema.TypeString,
-			Required:     true,
+			Optional:     true,
+			Computed:     true,
```

Comment:
In winbox, I can set this field. Are you sure it's computed?

Reply:
You're right, it should only be optional, not computed.